controllers/external-accounts.js

```diff
@@ -54,6 +54,7 @@ const syncExternalAccountData = async (req, res) => {
     const [discordUserData, rdsUserData] = await Promise.all([getDiscordMembers(), getDiscordUsers()]);
     const rdsUserDataMap = {};
     const updateUserDataPromises = [];
+    const userUpdatedWithInDiscordFalse = [];
```

Comment:
Nit: the name `userUpdatedWithInDiscordFalse` is too verbose. I would suggest `usersNotInDiscord` or `usersOutsideDiscord`

Reply:
This variable stores the users who are updated with in discord false.